src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/draft/DraftAdjudicationWorkflowController.kt

```diff
@@ -42,7 +43,6 @@ class DraftAdjudicationWorkflowController(
       ),
     ],
   )
-  @PreAuthorize("hasAuthority('SCOPE_write')")
   @ResponseStatus(HttpStatus.CREATED)
   fun completeDraftAdjudication(@PathVariable(name = "id") id: Long): ReportedAdjudicationDto =
```

Comment:
this is VIEW role. too

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976